js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties

```diff
@@ -177,3 +202,438 @@ category=Kategorie
+SSOSessionMaxRememberMe=SSO-Sitzung Maximaldauer "Angemeldet bleiben"
+ssoSessionMaxRememberMe=Maximale Zeitspanne, bevor eine Sitzung abläuft, wenn ein Benutzer die Option "Angemeldet bleiben" aktiviert hat. Token und Browser-Sitzungen werden ungültig, wenn eine Sitzung abläuft. Wenn nicht festgelegt, wird der Standardwert SSO-Sitzung Maximaldauer verwendet.
+clientSessionSettings=Einstellungen der Client-Sitzung
+clientSessionIdle=Client-Sitzung Leerlaufdauer
+clientSessionIdleHelp=Zeit, die eine Client-Sitzung inaktiv sein darf, bevor sie abläuft. Token werden ungültig, wenn eine Client-Sitzung abläuft. Die Option hat keine Auswirkungen auf die globale SSO-Benutzersitzung. Wenn sie nicht gesetzt ist, wird der Standardwert für SSO-Sitzung Leerlaufdauer verwendet.
+clientSessionMax=Client-Sitzung Maximaldauer
+clientSessionMaxHelp=Maximale Zeit, bevor eine Client-Sitzung abläuft. Token werden ungültig, wenn eine Sitzung abläuft. Die Option hat keine Auswirkungen auf die globale SSO-Benutzersitzung. Wenn sie nicht gesetzt ist, wird der Standardwert SSO-Sitzung Maximaldauer verwendet.
+offlineSessionSettings=Einstellungen für Offline-Sitzungen
+offlineSessionIdle=Offline-Sitzung Leerlaufdauer
+offlineSessionIdleHelp=Zeit, die eine Offline-Sitzung inaktiv sein darf, bevor sie abläuft. Sie müssen das Offline-Token mindestens einmal innerhalb dieses Zeitraums zur Aktualisierung verwenden; andernfalls läuft die Offline-Sitzung ab.
+offlineSessionMaxLimited=Offline-Sitzung Laufzeit Beschränkung
+offlineSessionMaxLimitedHelp=Erlaubt die Beschränkung der Maximalen Laufzeit einer Offline-Sitzung
+offlineSessionMax=Offline-Sitzung Maximaldauer
+offlineSessionMaxHelp=Maximale Zeit, bevor eine Offline-Sitzung unabhängig von der Aktivität abläuft.
+loginSettings=Anmeldeeinstellungen
+loginTimeout=Maximaldauer der Anmeldung
+loginTimeoutHelp=Maximale Zeit, die ein Benutzer zum Abschließen einer Anmeldung benötigt. Es wird empfohlen, diese Zeit relativ lang zu wählen, z. B. 30 Minuten oder mehr.
+loginActionTimeout=Maximaldauer eines Anmeldeaktion
+loginActionTimeoutHelp=Max. Zeit, die ein Benutzer hat, um anmeldebezogene Aktionen wie Passwort aktualisieren oder TOTP konfigurieren abzuschließen. Es wird empfohlen, diesen Wert relativ lang zu wählen, z. B. 5 Minuten oder mehr.
+defaultSigAlg=Standard Signatur Algorithmus
+defaultSigAlgHelp=Standardalgorithmus für das Signieren von Token für den Realm
+oAuthDeviceCodeLifespan=OAuth 2.0 Device Code Lebensdauer
+oAuthDeviceCodeLifespanHelp=Maximale Zeit bis zum Ablauf des Gerätecodes und des Benutzercodes. Dieser Wert muss lang genug sein, um nutzbar zu sein (so dass der Nutzer sein Zweitgerät abrufen, zur Verifizierungs-URI navigieren, sich anmelden usw.), sollte aber kurz genug sein, um die Nutzbarkeit eines für Phishing erlangten Codes zu begrenzen.
```

Comment:
"so dass der Nutzer sein Zweitgerät abrufen, zur Verifizierungs-URI navigieren, sich anmelden usw. KANN", "eines DURCH Phishing".

Reply:
Okay rewrote the sentence a bit:
```
oAuthDeviceCodeLifespanHelp=Maximale Zeit bis zum Ablauf des Gerätecodes und des Benutzercodes. Diese Dauer muss lang genug sein, so dass der Nutzer sein Zweitgerät abrufen, zur Verifizierungs-URI navigieren und sich anmelden kann. Die Dauer sollte aber kurz genug sein, um die Nutzbarkeit eines durch Phishing erlangten Codes zu begrenzen.
```